### DISCUSSION TITLE
Unable to set a height for the table

### DISCUSSION THREAD

**Discussion by natatotte** [mantinedev/mantine#6275] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.5.1

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

All

### Describe the bug

I'm trying to set a fixed height for the table in two ways. But both don't work, the table size remains fixed.

Setting the height directly in the <Table /> component:

```jsx
 <Table striped w="100%" stickyHeader withTableBorder verticalSpacing="xs" h="20rem">
                <Table.Thead>
                  <Table.Tr>
                    <Table.Th />
                    ...
```

Applying to the table root style:

```jsx
 <Table
                striped
                w="100%"
                stickyHeader
                withTableBorder
                verticalSpacing="xs"
                styles={{
                  table: {
                    height: '20rem'
                  }
                }}
              >
                <Table.Thead>
                  <Table.Tr>
                  ...
```

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
> I'm trying to set a fixed height for the table in two ways. But both don't work, the table size remains fixed.

I do not undertand this.

**Comment by natatotte** [comment]
> > I'm trying to set a fixed height for the table in two ways. But both don't work, the table size remains fixed.
> 
> I do not undertand this.

I'm trying to assign a height to my table. But it neither increases nor decreases in height. If I set a height to 0, it remains a fixed height.

**Comment by natatotte** [comment]
I added a Sandbox to reproduce

**Comment by rtivital** [comment]
Add a div element around the table and assing height there.
